source/clear-linux/guides/maintenance/swupd-search.rst

```diff
@@ -17,25 +17,26 @@ This guide assumes you:
 * Possess a basic knowledge of :ref:`swupd <swupd-guide>` 
 * Understand :ref:`how swupd differs <swupd-about>` from  
   other Linux\* distributions 
-* Plan to use :ref:`mixer` to build your own |CL| OS
+* May use :ref:`mixer` to build your own |CL| OS
 
 How do I search for a bundle? 
 *****************************
 
+Use `swupd search` to locate the bundle where the application binary exists. 
+
 Example: Kata Containers
 ========================
 
 Containers have revolutionized the way we manage cloud infrastructure. 
-Whereas traditional containers share the same OS kernel, raising security 
-concerns, with Kata Containers, each container has its own kernel instance 
-and runs on its own :abbr:`Virtual Machine (VM)`. Whether you're running 3 
-or 300 nodes on your cluster, Kata Containers provide a lightweight, fast, 
-and secure option for app/container management.  
+Traditional containers often share the same OS kernel, which raises 
+security concerns. Instead, with Kata Containers, each container has its own 
+kernel instance and runs on its own :abbr:`Virtual Machine (VM)`. Whether you're running 3 or 300 nodes on your cluster, Kata Containers provide a 
+lightweight, fast, and secure option for app/container management.  
 
-In |CL|, you only need to add one bundle to use `Kata Containers`_: 
-`containers virt`_. We also recommend our tutorial: :ref:`kata`
+In |CL|, you only need to add `this bundle`_ to use `Kata Containers`_: 
+`containers-virt`. Also, we recommend our tutorial: :ref:`kata`.
 
-We need a *kata* containers bundle. So how do we search for it? 
+We need to find *kata* containers in a bundle. How do we search for it? 
 
 #. Enter :command:`swupd search`, followed by 'kata' as the search term: 
```

Comment:
I would add something like  "Use the -b option to limit your search only designated binary paths or otherwise it will search everything"   Whenever I see guides and someone used a switch option I always wonder why they used that switch.

Reply:
OK ignore this comment. As I continue to read I see you defined it below.

Reply:
Point well taken. I was mulling this over last night. I will restate the broader search (without `-b`, first). Then I will include optional use of the `-b` flag. Minor tweak needed; will take care of soon.